spec/classes/maintenance/pg_repack_spec.rb

```diff
@@ -7,48 +7,47 @@
             .with_command('su - pe-postgres -s /bin/bash -c "/opt/puppetlabs/server/apps/postgresql/9.6/bin/pg_repack'\
-              ' -d pe-puppetdb --jobs 2 -t reports -t resource_events" > /var/log/puppetlabs/pe_databases_cron/output.log 2>&1')
+            ' -d pe-puppetdb --jobs 2 -t resource_events" > /var/log/puppetlabs/pe_databases_cron/output.log 2>&1')
         }
-      end
-      context 'on > PE 2019.1.0' do
-        before :each do
-          facts['pe_server_version'] = '2019.2.0'
-          facts['pe_postgresql_info']['installed_server_version'] = 9.6
-        end
         it {
           is_expected.to contain_cron('pg_repack reports tables')
             .with_command('su - pe-postgres -s /bin/bash -c "/opt/puppetlabs/server/apps/postgresql/9.6/bin/pg_repack'\
-              ' -d pe-puppetdb --jobs 2 -t reports -t resource_events" > /var/log/puppetlabs/pe_databases_cron/output.log 2>&1')
+            ' -d pe-puppetdb --jobs 2 -t reports" > /var/log/puppetlabs/pe_databases_cron/output.log 2>&1')
         }
       end
-      context 'on < PE 2019.1.0' do
+      context 'on < PE 2019.3.0 with postgresql 11' do
         before :each do
-          facts['pe_server_version'] = '2019.0.0'
-          facts['pe_postgresql_info']['installed_server_version'] = 9.6
+          facts['pe_server_version'] = '2019.2.0'
+          facts['pe_postgresql_info']['installed_server_version'] = 11
         end
+        it {
+          is_expected.to contain_cron('pg_repack resource_events tables')
+            .with_command('su - pe-postgres -s /bin/bash -c "/opt/puppetlabs/server/apps/postgresql/11/bin/pg_repack'\
+              ' -d pe-puppetdb --jobs 2 -t resource_events" > /var/log/puppetlabs/pe_databases_cron/output.log 2>&1')
+        }
         it {
           is_expected.to contain_cron('pg_repack reports tables')
-            .with_command('su - pe-postgres -s /bin/bash -c "/opt/puppetlabs/server/apps/postgresql/bin/pg_repack'\
+            .with_command('su - pe-postgres -s /bin/bash -c "/opt/puppetlabs/server/apps/postgresql/11/bin/pg_repack'\
               ' -d pe-puppetdb --jobs 2 -t reports" > /var/log/puppetlabs/pe_databases_cron/output.log 2>&1')
         }
       end
-      context 'on > PE 2019.1.0 with postgresql 11' do
+      context 'on < PE 2019.7.0' do
         before :each do
-          facts['pe_server_version'] = '2019.2.0'
+          facts['pe_server_version'] = '2019.5.0'
           facts['pe_postgresql_info']['installed_server_version'] = 11
         end
         it {
           is_expected.to contain_cron('pg_repack reports tables')
             .with_command('su - pe-postgres -s /bin/bash -c "/opt/puppetlabs/server/apps/postgresql/11/bin/pg_repack'\
-              ' -d pe-puppetdb --jobs 2 -t reports -t resource_events" > /var/log/puppetlabs/pe_databases_cron/output.log 2>&1')
+              ' -d pe-puppetdb --jobs 2 -t reports" > /var/log/puppetlabs/pe_databases_cron/output.log 2>&1')
         }
       end
     end
```

Comment:
We are missing a test for 2019.7.0+ that confirms that it does not `repack` either reports or `resource_events`.